### DISCUSSION TITLE
Incorrect/buggy type inlay for Multiline Strings

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/pylance-release#3245] [Enhancement]
Pylance bugs out when producing inlays for a multiline string.

## Environment data

-   Language Server version: Pylance language server 2022.8.40 (pyright f444ffa5)
-   OS and version: MacOS Monterey (12.4)
-   Python version (& distribution if applicable, e.g. Anaconda): 3.7.9 using venv

## Code Snippet

![image](https://user-images.githubusercontent.com/98395146/186593465-1ab6611f-1bdd-428e-a498-f7cdaf695be5.png)

## Repro Steps

Try using putting some string formatting inserts inside a multiline string

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/98395146/186594471-8763d313-3ee7-41bd-9a60-edd492e81663.png)
A better explaination

**Comment by bschnurr** [comment]
currently we limit hint length and append '...' when that limit is reached.

**Reply by ghost** [reply]
I think its better to have it return just `Literal String` for multiline strings

**Comment by bschnurr** [comment]
I think we might be able to add a 'onHover' trigger to show the entire hint when you hover it.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
